app/navigation/Navigator.scala

```diff
@@ -468,7 +462,7 @@ class Navigator @Inject()(implicit frontendAppConfig: FrontendAppConfig)
   private def annualPayAmountRoutes: UserAnswers => Call =
     userAnswers =>
       if (isPhaseTwoOnwards(userAnswers)) {
```

Comment:
I know it's fine as is already just wondering if it's worth re writing the nested if else's into a match case 
```
  private def annualPayAmountRoutes: UserAnswers => Call = { userAnswers =>
    {
      (isPhaseTwoOnwards(userAnswers), isMayExtensionOnwards(userAnswers)) match {
        case (true, true)  => routes.HasEmployeeBeenOnStatutoryLeaveController.onPageLoad()
        case (true, false) => routes.PartTimeQuestionController.onPageLoad()
        case _             => phaseOneAnnualPayAmountRoute(userAnswers)
      }
    }
  }
```

Reply:
Good shout that's easier to read